src/handlers/sendEmailRequestToNotify.ts

```diff
@@ -0,0 +1,41 @@
+import { APIGatewayProxyEvent } from 'aws-lambda'
+import { NotifyClient } from 'notifications-node-client'
+import { retrieveNotifySecrets } from '../secrets/retrieveNotifySecrets'
+import { CustomAxiosResponse } from '../types/notify/axiosResponse'
+import { tryParseJSON } from '../utils/helpers'
+
+// event type is a placeholder
+export const handler = async (event: APIGatewayProxyEvent) => {
+  if (!event.body) {
+    console.log('Could not find event body. An email has not been sent')
+    return
+  }
+
+  const requestDetails = tryParseJSON(event.body)
+  const secrets = await retrieveNotifySecrets()
+  const notifyClient = new NotifyClient(secrets.notifyApiKey)
+
+  console.log('Sending request to Notify')
+  try {
+    const response = (await Promise.resolve(
+      notifyClient.sendEmail(secrets.notifyTemplateId, requestDetails.email, {
+        personalisation: {
+          firstName: requestDetails.firstName,
+          zendeskId: requestDetails.zendeskId,
+          signedUrl: requestDetails.signedUrl
+        }
+      })
+    )) as unknown as CustomAxiosResponse
```

Comment:
Not a huge fan of this double-cast, wonder if there's a way to do it without this?